src/script/api/game_changelog.hpp

```diff
@@ -21,6 +21,14 @@
  * \li GSNewGRF
  * \li GSNewGRFList
  *
+ * API removals:
+ * \li GSWindow::NetworkStartServerWidgets::WID_NSS_SPECTATORS_LABEL
```

Comment:
The GUI changes with every version. GSWindow explicitly states that it has no version guarantee. We never document GUI changes here.